resident-services/scripts/Resident_Test_script.jmx

```diff
@@ -29,21 +29,6 @@
             <stringProp name="Argument.value">443</stringProp>
             <stringProp name="Argument.metadata">=</stringProp>
           </elementProp>
-          <elementProp name="relyingPartyId" elementType="Argument">
-            <stringProp name="Argument.name">relyingPartyId</stringProp>
-            <stringProp name="Argument.value">Bharathi-Inc</stringProp>
-            <stringProp name="Argument.metadata">=</stringProp>
-          </elementProp>
-          <elementProp name="userClaims" elementType="Argument">
-            <stringProp name="Argument.name">userClaims</stringProp>
-            <stringProp name="Argument.value">&quot;name&quot;,&quot;gender&quot;,&quot;birthdate&quot;,&quot;email&quot;,&quot;phone_number&quot;</stringProp>
-            <stringProp name="Argument.metadata">=</stringProp>
-          </elementProp>
-          <elementProp name="userClaimsUpdateOIDC" elementType="Argument">
```

Comment:
I see Uri is in mixed case, can we change everywhere to URI ?

Reply:
Done